README.md

```diff
@@ -1,86 +1,104 @@
 # Biometry Service
 
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE) ![Version](https://img.shields.io/nuget/v/BiometryService?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/BiometryService?style=flat-square)
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+This library offers a simple contract to use the biometry across Android, iOS and Windows (UWP & WinUI).
 
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+## Getting Started
 
-This library offers a simple contract to use the biometry across Android, iOS and Windows (UWP & WinUI).
+1. Install the latest stable version of `BiometryService` in your platform heads and `BiometryService.Abstractions` in your presentation layer if you are using MVVM pattern, and if not just install both in your platform heads.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+   
+1. Instanciate BiometryService.
 
-## Features
+   ### UWP/Windows
```

Comment:
Use either "WinUI/UWP" or simply "Windows" (if the code is the same).
Remember that UWP is being deprecated so WinUI should always be mentioned first.